common/src/nexus_config.rs

```diff
@@ -351,7 +354,7 @@ pub struct PackageConfig {
     pub tunables: Tunables,
     /// `Dendrite` dataplane daemon configuration
     #[serde(default)]
-    pub dendrite: DpdConfig,
+    pub dendrite: HashMap<Name, DpdConfig>,
```

Comment:
`SwitchLocation` wasn't a thing yet during this change, IIRC. Pretty much everything has been updated to `SwitchLocation` in the commit I'm currently working on.

The `HashMap` is setting us up for having any number of switches (i.e. multi-rack), although `SwitchLocation` will likely need to be extended to allow specifying Rack & Slot, instead of just Slot.

Reply:
Also, the `HashMap` isn't getting exercised in what you currently see, but there's more to come :)